common/network-common/src/test/java/org/apache/spark/network/TransportClientFactorySuite.java

```diff
@@ -117,7 +117,7 @@ private void testClientReuse(int maxConnections, boolean concurrent)
       }
 
       Assert.assertEquals(0, failed.get());
-      Assert.assertEquals(clients.size(), maxConnections);
+      Assert.assertTrue(clients.size() <= maxConnections);
```

Comment:
Could you update the PR description if we have a previous Jenkins test failure for this, please?